x-pack/plugins/apm/public/components/shared/ImpactBar/index.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiProgress } from '@elastic/eui';
+import React from 'react';
+
+interface Props extends StringMap<any> {
+  value: any;
+}
+
+export function ImpactBar({ value, ...rest }: Props) {
+  return (
+    <EuiProgress size="l" value={value} max={100} color="danger" {...rest} />
```

Comment:
@jasonrhodes you're right that the Impact is important and most likely is a cause for attention and worry, but I think I might prefer having this in a more subtle colour since it's not the most important element on the page. I agree that we shouldn't go back to the calm blue or even affirmative green ("all is good"), so perhaps changing it to `warning` is worth the try. The blue colour that was there was a legacy design from the Opbeat days.

Reply:
@formgeist I'll try warning and post the screenshot -- if that still seems too aggressive esp with it being on every row, we could use one of the greys

Reply:
@jasonrhodes awesome, let's check it out and take it from there 👍

Reply:
@formgeist so for some reason, EUIProgress doesn't work with "warning" (possibly an EUI bug) -- for now these look like the choices. I've left it with "subdued" (gray) for now but lmk if you like another one here:
<img width="1800" alt="screen shot 2018-09-04 at 12 08 43 pm" src="https://user-images.githubusercontent.com/159370/45043507-51fe9d00-b03b-11e8-8a39-9246fa614b4f.png">
<img width="1803" alt="screen shot 2018-09-04 at 12 07 33 pm" src="https://user-images.githubusercontent.com/159370/45043508-51fe9d00-b03b-11e8-8651-6566c72088a8.png">
<img width="1796" alt="screen shot 2018-09-04 at 12 07 10 pm" src="https://user-images.githubusercontent.com/159370/45043509-51fe9d00-b03b-11e8-8276-c204dde4bf52.png">

Reply:
I prefer blue but I probably just have Stockholm Syndrome from the Opbeat days

Reply:
db0599ba01

Reply:
I reckon we go with `primary`, the blue that we're used to. Sorry for having to go over this again !